src/test/cls_rbd/test_cls_rbd.cc

```diff
@@ -84,10 +85,35 @@ class TestClsRbd : public ::testing::Test {
 
 };
 
+class TestClsRbdCg : public ::testing::Test {
```

Comment:
Existing test harness doesn't clean up after every test. It's convenient to have a clean pool before the test.